Components/Grid/grid.css

```diff
@@ -0,0 +1,22 @@
+.grid-section{
+    margin: -2rem auto auto 4rem;
```

Comment:
Any specific reasons for using `-2rem` here?

Reply:
No no, I just overlooked that and fixed it now, Thanks for pointing it out 👍🏻